docs/guides/k8s-ha-monitoring-via-vm-cluster.md

```diff
@@ -234,7 +234,7 @@ scrape_configs:
 ```
 
 * `remoteWriteUrls: - http://vmcluster-victoria-metrics-cluster-vminsert.default.svc.cluster.local:8480/insert/0/prometheus/` configures `vmagent` to write scraped metrics to the `vmselect service`.
-* The `metric_ralabel_configs` section allows to process Kubernetes metrics for the Grafana dashboard.
+* The `metric_ralabel_configs` section allows you to process Kubernetes metrics for the Grafana dashboard.
```

Comment:
there is no sense in saying "you" or "your" unless it is really needed. It sounds a bit rude